cmd/clusterctl/cmd/delete.go

```diff
@@ -92,6 +94,11 @@ func init() {
 	deleteCmd.Flags().StringVar(&dd.kubeconfigContext, "kubeconfig-context", "",
 		"Context to be used within the kubeconfig file. If empty, current context will be used.")
 
+	deleteCmd.Flags().Float32Var(&dd.restQPS, "rest-qps", 0,
```

Comment:
q: should the default value be 20, so we are explicit about what we are actual using if the user do not set the flag
(similar comment apply to burst)

Reply:
This 0 does get transformed into the "real" default later on, but yes it would be simple to define the default here too. I'll make that change.

Reply:
Thanks, we can use the same const
(this is printed in the command help)